roles/cinder-common/meta/main.yml

```diff
@@ -22,3 +22,4 @@ dependencies:
     when: openstack_install_method == 'package'
   - role: ceph-common
     when: ceph.enabled|bool
+    role: serverspec
```

Comment:
This needs to be `- role: serverspec`. The role serverspec is not currently part of the yaml list.